app/controllers/BackLinkModel.scala

```diff
@@ -138,6 +138,10 @@ class BackLinkModel @Inject() (
         Some(PendingPaymentController.loadPendingPaymentPage)
       case "add"                                                                          =>
         Some(DashboardController.showDashboard)
+      case x // other goods has cancel button that is because it is following different pattern of adding items
```

Comment:
suggestion make this case val something like `case selectGoods if path.endswi..` (?)

Should we include both back and cancel on other goods since they said add to all or that causes another issue ?

Reply:
not sure let me check this with design team, and come back

Reply:
please check comments on jiras, for all the details after meeting with design team